### PR TITLE
[git-webkit] Mark landed changes as merged

### PR DESCRIPTION
**Pull request by JonWBedard**
#### 55b48cedb2921099811dd3cc5c20f563377eac82
<pre>
[git-webkit] Mark landed changes as merged
<a href="https://bugs.webkit.org/show_bug.cgi?id=233056">https://bugs.webkit.org/show_bug.cgi?id=233056</a>
&lt;rdar://problem/85351564 &gt;

Reviewed by Dewei Zhu.

For BitBucket and GitHub to recognize a pull-request as merged, the target
branch must be updated with the exact commit to be merged as the merge occurs.

* Scripts/libraries/webkitscmpy/setup.py: Bump version.
* Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Ditto.
* Scripts/libraries/webkitscmpy/webkitscmpy/program/land.py:
(Land.main): Update source branch when merging commits to target branch.
* Scripts/libraries/webkitscmpy/webkitscmpy/remote/bitbucket.py:
(BitBucket.PRGenerator.update): Only delete values if they exist.
* Scripts/libraries/webkitscmpy/webkitscmpy/test/land_unittest.py:
(TestLand):
(TestLandGitHub):
(TestLandBitBucket):

Canonical link: <a href="https://commits.webkit.org/244312@main">https://commits.webkit.org/244312@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@285890">https://svn.webkit.org/repository/webkit/trunk@285890</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

### CODE REVIEW

**Comment by JonWBedard** [comment]
For reference, https://github.com/WebKit/WebKit/pull/35 was landed with this technique to prove that it works for git-svn

File: Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/bitbucket.py [hunk 1]
```diff
@@ -202,8 +202,10 @@ def update(self, pull_request, head=None, title=None, body=None, commits=None, b
             if response.status_code // 100 != 2:
                 return None
             data = response.json()
-            del data['author']
-            del data['participants']
+            if 'author' in data:
```
Comment (dewei-zhu) [inline Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/bitbucket.py]: Nit: `data.pop('author', None)`?

**Review by dewei-zhu — APPROVED** [review]
r=me

File: Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/bitbucket.py (hunk 1 above)
Comment (JonWBedard) [reply Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/bitbucket.py]: Good idea!

**Comment by JonWBedard** [comment]
Landed https://commits.webkit.org/244312@main (55b48cedb2921099811dd3cc5c20f563377eac82)!

**Pull request merged**